creme/creme_config/static/creme_config/js/widgets.js

```diff
@@ -0,0 +1,60 @@
+
+(function($) {
+    "use strict";
+
+    function buildButtonChoice(choice) {
+        return $("<span>").addClass("menu_button").text(choice.label).attr('title', choice.description).append(
+            $("<input>").attr({
+                type: "hidden",
+                name: choice.name
+            }).val(choice.value).prop("disabled", !choice.selected)
+        );
```

Comment:
There is a String.prototype.template method in creme similar to an extremely simplified es6 one.
```suggestion
return $((
    '<span class="menu_button" title="${description}">' +
        '${label}<input type="hidden" name="${name} val="${value}"/>' +
    '</span>'
).template(choice)).prop("disabled", !choice.selected);
```